RUFAS/input_manager.py

```diff
@@ -2278,6 +2279,64 @@ def _create_record(self, data_entry: Dict[str, Any], name: str) -> Dict[str, Any
             "minimum": data_entry.get("minimum", ""),
         }
 
+    def compare_metadata_properties(self, properties_file_path: Path, comparison_properties_file_path: Path) -> None:
+        """
+        Compares two metadataa properties json files using the DeepDiff package and saves the results in a txt file.
+        """
+        info_map = {
+            "class": self.__class__.__name__,
+            "function": self.compare_metadata_properties.__name__,
+        }
+        self._load_metadata(properties_file_path)
+        data1 = deepcopy(self.meta_data)
+        self.meta_data = {}
+        self._load_metadata(comparison_properties_file_path)
+        data2 = deepcopy(self.meta_data)
+
+        diff = DeepDiff(data1, data2, ignore_order=True, verbose_level=2)
+
+        file_name = (
+            "diff_results_"
+            + os.path.basename(str(properties_file_path))
+            + "_vs_"
+            + os.path.basename(str(comparison_properties_file_path))
+        )
```

Comment:
```suggestion
        first_file_path = os.path.basename(str(properties_file_path))
        second_file_path = os.path.basename(str(comparison_properties_file_path))
        file_name = f"diff_results_{first_file_path}_vs_{second_file_path}"
```
I think f-strings make this a little cleaner.

Reply:
Much better, thank you